common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -341,92 +240,74 @@ nav.sequence-nav {
-        margin-bottom: 0;
-
-        a {
-          background-position: center center;
-          background-repeat: no-repeat;
-          border: none;
-          display: block;
-          padding: lh(0.5) 4px;
-          text-indent: -9999px;
-          overflow: hidden;
-          @include transition(all .2s $ease-in-out-quad 0s);
-
-          &:hover, &:focus {
-            opacity: 0.5;
-            background-position: center 15px;
-          }
-
-          &.disabled {
-            opacity: 0.4;
-          }
-
-/*          &:focus {
-            outline: 0;
-          }
-*/        }
-      }
+.sequence-nav-button {
+  @include linear-gradient(top, #eee, #ddd);
+  @include transition(all .2s $ease-in-out-quad 0s);
+  position: absolute;
+  top: 0;
+  z-index: 50;
+  display: block;
+  height: 44px;
+  width: ($baseline*2);
+  border: 1px solid $gray-l3;
+  box-shadow: 0 1px 0 rgba(255, 255, 255, .7) inset;
+  background-position: center;
+  background-repeat: no-repeat;
+  text-indent: -9999px;
+  overflow: hidden;
+
+  &.prev {
+    @include border-radius(35px, 0, 0, 35px);
+    @include left(0);
+    background-position: center 15px;
+
+    // CASE: left to right layout
+    @include ltr {
+      background-image: url('../images/sequence-nav/previous-icon.png');
+    }
 
-      &.prev {
-        @include border-radius(35px, 0, 0, 35px);
+    // CASE: right to left layout
+    @include rtl {
+      background-image: url('../images/sequence-nav/next-icon.png');
+    }
+  }
 
-        a {
-          background-position: center 15px;
+  &.next {
+    @include border-radius(0, 35px, 35px, 0);
+    @include right(0);
+    background-position: center 15px;
 
-         // CASE: left to right layout
-         @include ltr {
-            background-image: url('../images/sequence-nav/previous-icon.png');
-          }
+    // CASE: left to right layout
+    @include ltr {
+      background-image: url('../images/sequence-nav/next-icon.png');
+    }
 
-          // CASE: right to left layout
-          @include rtl {
-            background-image: url('../images/sequence-nav/next-icon.png');
-          }
-        }
-      }
+    // CASE: right to left layout
+    @include rtl {
+      background-image: url('../images/sequence-nav/previous-icon.png');
+    }
+  }
 
-      &.next {
-        @include border-radius(0, 35px, 35px, 0);
-        @include border-left(none);
+  &:hover, &:focus {
+    opacity: 0.5;
+  }
 
-        a {
-          background-position: center 15px;
+  &.disabled {
+    cursor: normal;
+    opacity: 0.4;
+  }
+}
 
-          // CASE: left to right layout
-          @include ltr {
-            background-image: url('../images/sequence-nav/next-icon.png');
-          }
+.seq_contents {
+  display: none;
+}
 
-          // CASE: right to left layout
-          @include rtl {
-            background-image: url('../images/sequence-nav/previous-icon.png');
-          }
-        }
-      }
-    }
-  }
+.sequence-bottom {
```

Comment:
@clrux, this looks like a clean up of the existing rule (https://github.com/edx/edx-platform/pull/6739/files#diff-9196fe3ebe3e4a65e923c197a2999fb8L348). The styles don't seem to match up completely. Did you add anything new here or tweak anything?